src/System.Xml.XPath.XDocument.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30723.0
+# Visual Studio 14
+VisualStudioVersion = 14.0.22310.1
```

Comment:
@krwq @weshaggard  It has not been my experience that VS actually does this every time. I don't like having a mismatch of these nor noise in the history. It can still open in VS 2013, but this controls which version is used by default when you double click when you have both.

I would have reverted it. This is not your typical VS reformatting argument for say poor  csproj whitespace -- it's a semantic change.